migrations/20220428092250_initial_tables.js

```diff
@@ -17,6 +17,23 @@ exports.up = async (knex) => {
 
     def.primary(['id'])
   })
+
+  await knex.schema.createTable('recipe', (def) => {
+    def.uuid('id').defaultTo(uuidGenerateV4())
+
+    def.datetime('created_at').notNullable().default(now())
+    def.datetime('updated_at').notNullable().default(now())
+    def.string('externalId')
+    def.string('name')
+    def.string('imageAttachmentId')
+    def.string('material')
+    def.string('alloy')
+    def.string('price')
+    def.jsonb('requiredCerts') // TODO
+    def.string('supplier')
+
+    def.primary(['id'])
+  })
 }
 
 exports.down = async (knex) => {
```

Comment:
The table needs to be dropped in the exports.down function for when things like rollbacks happen.

Reply:
`up`/`down` scripts is a farily established standard. It's extremely useful for dev testing imho and occassionally has use elsewhere. This is also the pattern of `knex`